src/module.ts

```diff
@@ -67,7 +67,7 @@ export default defineNuxtModule<ModuleOptions>({
   },
   async setup(options, nuxt) {
     
-    if(process.env.__STORYBOOK__)
+    if(import.meta.env.__STORYBOOK__)
```

Comment:
This line seems to be causing a bug for me when I attempt to install the latest version of `@nuxtjs/storybook`, pinpointed the issue down to the tagged release this was included in:

![image](https://github.com/nuxt-modules/storybook/assets/6605678/8ef5113b-9f04-4f70-8bd6-f575074bce5c)


Getting the error when I attempt to install via `yarn add @nuxtjs/storybook@latest`